etc/Dockerfile.floonet

```diff
@@ -37,13 +37,13 @@ COPY --from=builder /usr/src/grin/target/release/grin /usr/local/bin/grin
 
 WORKDIR /root/.grin
 
-RUN grin --floonet server config && \
+RUN grin --testnet server config && \
```

Comment:
File itself needs renaming here I think as we now reference it via the new `testnet` suffix.